src/main/kotlin/domain/CalculatorType.kt

```diff
@@ -0,0 +1,13 @@
+package domain
+
+enum class CalculatorType(val mathematicalExpression: (Double, Double) -> Double) {
+    ADDITION({ a, b -> kotlin.math.round((a + b) * domain.Calculator.Companion.ROUND) / domain.Calculator.Companion.ROUND }),
+    SUBTRACT({ a, b -> kotlin.math.round((a - b) * domain.Calculator.Companion.ROUND) / domain.Calculator.Companion.ROUND }),
```

Comment:
```suggestion
enum class CalculatorType(val type: String, val mathematicalExpression: (Double, Double) -> Double) {
    ADDITION("+", { a, b -> a + b),
    SUBTRACT("-", { a, b -> a - b)
```
CalculatorType 에서 어떤 타입인지의 문자 값을 가지는 방법도 좋을것 같아요
추가로 로직의 복잡도를 낮추고 간단하게 구현해보세요 😄